api/desecapi/models.py

```diff
@@ -280,6 +280,11 @@ def _partitioned_name(self):
         subname, _, parent_name = self.name.partition('.')
         return subname, parent_name or None
 
+    def get_catalog_rrset(self, delete=False):
```

Comment:
This code may be pdns specific and shouldn't live in models.py

Reply:
fix in 2f649f29870a7008e78608be28d0997ba20c0a98